pandas-stubs/core/groupby/generic.pyi

```diff
@@ -101,7 +103,34 @@ class SeriesGroupBy(GroupBy):
     def nlargest(self, n: int = ..., keep: str = ...) -> Series[S1]: ...
     def nsmallest(self, n: int = ..., keep: str = ...) -> Series[S1]: ...
     def nth(self, n: int | Sequence[int], dropna: str | None = ...) -> Series[S1]: ...
-    def sum(self, **kwargs) -> Series[S1]: ...
+    def sum(
+        self,
+        numeric_only: bool = ...,
+        min_count: int = ...,
+        engine=...,
+        engine_kwargs=...,
+    ) -> Series[S1]: ...
+    def prod(self, numeric_only: bool = ..., min_count: int = ...) -> Series[S1]: ...
+    def sem(self, ddof: int = ...) -> Series[S1]: ...
+    def std(self, ddof: int = ...) -> Series[S1]: ...
+    def var(self, ddof: int = ...) -> Series[S1]: ...
```

Comment:
I think that `sem`, `std` and `var` have to return `Series[float]` because they are numerical calculations.